CHANGES.rst

```diff
@@ -0,0 +1,7 @@
+Changelog
+=========
+
+0.6.2 (unreleased)
+----------------
```

Comment:
Oh, right, I completely forgot that ZODB3 was an empty transitional package these days.